dll/win32/setupapi/setupapi_private.h

```diff
@@ -383,4 +383,9 @@ LPSTR WINAPI UnicodeToMultiByte(LPCWSTR lpUnicodeStr, UINT uCodePage);
 typedef BOOL (*FIND_CALLBACK)(LPCWSTR SectionName, PVOID Context);
 BOOL EnumerateSectionsStartingWith(HINF hInf, LPCWSTR pStr, FIND_CALLBACK Callback, PVOID Context);
 
+PVOID* srclist_temporary_sources;
+UINT srclist_temporary_sources_count;
+static BOOLEAN noBrowse = FALSE;
```

Comment:
Please move these declarations at the top of the file (see the other externs).
Also, this is not the correct place where to define noBrowse.
See `GlobalSetupFlags` for an example.